guides/v2.0/coding-standards/technical-guidelines.md

```diff
@@ -528,6 +532,10 @@ class View extends Template
 
 We are reviewing this section and will publish it soon.
 
+6.4.1. Domain/business logic MUST be executed on the Service Contracts layer
```

Comment:
Should we add "Domain layer"? Consider it when adding "Service Layer" section. These rules can be moved there then.